package.json

```diff
@@ -37,7 +37,9 @@
   "workspaces": [
     "packages/*",
     "tools/*",
-    "!packages/helloworld"
+    "!packages/helloworld",
+    "!packages/hermes-inspector-msggen",
+    "!packages/react-native-bots"
```

Comment:
My only issue is it's hard to add diff tags so toss might accidentally get undone in the future